bindata/network/additional-networks/cr/cr.yaml

```diff
@@ -3,6 +3,10 @@ kind: NetworkAttachmentDefinition
 metadata:
   name: {{.AdditionalNetworkName}}
   namespace: default
+{{if .ConfigAnnotations}}
+  annotations:
+    k8s.v1.cni.cncf.io/resourceName: openshift.com/sriov
```

Comment:
Shouldn't this be `openshift.io/sriov`?

Reply:
Functionally it doesn't matter whether the resourceName suffix is ``com`` or ``io``,  but I agree that ``openshift.io`` would be more appropriate name used in operator, going to make the change.